html.css

```diff
@@ -0,0 +1,180 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;700&display=swap');
```

Comment:
- [ ] Please, consider following general requirements:
  - Pull Request should have a descriptive title telling you what changes are inside.
  - Pull Request should have a short summary describing introduced changes in more detail.
<img width="968" alt="Screen Shot 2022-05-26 at 09 58 39" src="https://user-images.githubusercontent.com/49064106/170427223-ad6e8001-a656-41a4-a45a-589cf472df7a.png">

Reply:
![image](https://user-images.githubusercontent.com/13853450/170431170-9dc63f94-226f-44a7-a71b-6261df96e466.png)
Thanks, Done the changes.

Reply:
👍